src/components/MultipleAtropos.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/function-component-definition */
```

Comment:
Instead of adding this comment everywhere just disable it once in eslintconfig file